### PR TITLE
Add new Westeros tutorial: depicting renewable resource supply curves

### PR DESCRIPTION
**Pull request by OFR-IIASA**
This PR adds a tutorial on how to model renewable resource supply curves.

This is the final of three tutorials on various features surrounding the renewables.

## How to review

Run tutorial
Read through tutorials and see if the explanantions are understandable 

## PR checklist

~ - [ ] Tests added. Tutorial added~
- [x] Documentation added.
- [x] Release notes updated.

### CODE REVIEW

**Comment by khaeru** [comment]
Postpone to v3.3, as indicated by @OFR-IIASA on 2021-01-13.

**Review by behnam-zakeri — COMMENTED** [review]
@OFR-IIASA, thank you for this useful tutorial. Please consider the following changes/improvements:
1. As you have mentioned "supply curve", it will be nice to see that curve. So, if you could make a two-dimensional chart, one showing "potential" and another one showing "capacity factor", will be useful.
2. This is a suggestion, but up to you implement or not. At the moment, the model picks the renewable potential with the highest capacity factor first, and when saturated, the model picks the next highest capacity factor. It may be more interesting to add cost to these potentials, meaning that, a high potential windy area may not be necessarily the first area to harvest wind, because it may be far from the grid. I suggest adding one technology for each grade, called something like "connection_to_grid", with some sort of cost (simply variable cost and no vintaging), would be useful to make the example even more educative and realistic.
3. Possibly apply an intuitive and short heading, sub-headings with logical size (level 1 bigger than level 2 etc.), that can help to navigate through the tutorial.
4. Reviewing the units of capacity factor parameter. At the moment it is % but numbers are lower than 1.

File: tutorial/westeros/westeros_renewable_resource.ipynb [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+    "The capacity factor of the `wind_ppl` will remain unchanged and will be reflected in the parametrization of the `renewable_resources`.  \n",
+    "\n",
+    "The following steps are required:\n",
+    "1. Add level and commodity:\n",
+    "   - Specify and new level and commodity which accounts for the wind potentials and which serve as inputs to the `wind_ppl`\n",
+    "   - Specify which level is a `level_renewable`\n",
+    "2. Modify existing renewable technology:\n",
+    "   - Specify which technology is classified as a `type_tec` renewable (optional) \n",
+    "   - Modify the input of the `wind_ppl`\n",
+    "3. Add potentials and corresponding \n",
```
Comment (behnam-zakeri) [inline tutorial/westeros/westeros_renewable_resource.ipynb:117]: It seems this sentence needs more information after "corresponding"?

**Comment by behnam-zakeri** [comment]
Thanks @OFR-IIASA for addressing the remaining issues. The idea of the exercise for the users is very nice. There is a small missing sentence I highlighted in an in-line comments. Other than, it's ready to merge.

**Comment by behnam-zakeri** [comment]
Thanks @OFR-IIASA and @khaeru. This PR is ready to merge.

**Review by khaeru — APPROVED** [review]
Thanks both. I'll rebase this to resolve the conflict in the tutorials README and clean up the [commit messages](https://chris.beams.io/posts/git-commit/#seven-rules).

FYI, I've made a improvement in #476 that will, I hope, mean that we will no longer need to rebase to resolve conflicts in RELEASE_NOTES.rst.

**Comment by khaeru** [comment]
Incidentally, the branch history seems clean already, @OFR-IIASA, so whatever rebasing you did previously seems to have worked :+1:. See at the top it says "Commits: 8" under the PR title.

**Pull request merged**